flask_common/mongo/documents.py

```diff
@@ -127,8 +127,8 @@ def delete(self, **kwargs):
             self.modify(set__is_deleted=self.is_deleted)
 
     @queryset_manager
-    def all_objects(doc_cls, queryset):
-        if not hasattr(doc_cls, '_all_objs_queryset'):
+    def all_objects(self, queryset):
```

Comment:
This is a false positive imho. Queryset managers are not instances of documents. Maybe turn off this specific rule?

Reply:
Thanks for the suggestion.

Reply:
Looks like I turned off all name rules as recommended by black. I will undo this change.

Reply:
I don't think black has anything to do with `self` naming, so it's just a weird mongoengine interface.